src/main/java/org/example/android/ProductPage.java

```diff
@@ -32,11 +32,11 @@ public class ProductPage extends ProductPageBase {
     @AndroidFindBy(xpath = "//android.widget.TextView[@text='PRODUCTS']")
     private ExtendedWebElement pageTitle;
 
-    @AndroidFindBy(xpath = "//android.view.ViewGroup[@content-desc='test-Item']")
+    @AndroidFindBy(xpath = "//android.view.ViewGroup[contains(@content-desc, 'test-Item')]")
     private List<ProductListItemComponent> productItems;
 
-    @AndroidFindBy(accessibility = "test-Item title")
-    private ExtendedWebElement detailsTitle;
+    @AndroidFindBy(xpath = "//android.widget.ScrollView//android.widget.TextView[contains(@text, '.')]")
```

Comment:
if this price  it should be    @ExtendedFindBy(accessibilityId = "test-Price") on ProductListItemComponent